frontend/packages/kubevirt-plugin/src/k8s/requests/cdi-upload/cdi-upload-requests.ts

```diff
@@ -67,6 +67,11 @@ const createUploadToken = async (pvcName: string, namespace: string): Promise<st
 export const createUploadPVC = async (dataVolume: V1alpha1DataVolume) => {
   const dataVolumeName = getName(dataVolume);
   const namespace = getNamespace(dataVolume);
+  // BZ-1915694: uploaded DV must have "storage.bind.immediate.requested" annotation.
+  dataVolume.metadata.annotations = {
+    ...dataVolume.metadata.annotations,
+    'storage.bind.immediate.requested': 'true',
```

Comment:
can you create a const for it in `src/components/cdi-upload-provider/consts.ts`

Reply:
sure +1